c++/basicio.hpp

```diff
@@ -177,6 +195,16 @@ inline auto read_matrix_text(const std::string &filename, const bool verbose = f
   return read_matrix_data(F, [](auto &F) { return read_one<double>(F); }, dim1, dim2);
```

Comment:
Hmm, ta interface je pravzaprav storast. Kaj ce bi za F naredili capture v lambdi, potem F niti ni potreben.
Torej *read_matrix_data(FNC next_value, ...)
in spodaj
 [ &F ] () { return read_one<double>(F); }
in
 [ &F ] () { double x; F.read  etc. etc. }

Reply:
* ni mišljeno kot pointer ne? Nisem točno ziher, kaj je smisel tega. Da lahko podaš bolj splošno funkcijo, ki ni nujno povezana s fileom?

Reply:
"*"

Reply:
Morda to kar pusti, bom jaz naredil, ker imam namen spremeniti se nakaj drugih stvari.